tests/test_driver/firebase_app_check/firebase_app_check_e2e.dart

```diff
@@ -20,48 +19,41 @@ void setupTests() {
         );
       });
 
-      // test('activate', () async {
-      //   await expectLater(
-      //     FirebaseAppCheck.instance.activate(
-      //       webRecaptchaSiteKey: '6Lemcn0dAAAAABLkf6aiiHvpGD6x-zF3nOSDU2M8',
-      //     ),
-      //     completes,
-      //   );
-      // });
-
-      // test(
-      //   'getToken',
-      //   () async {
-      //     final token = await FirebaseAppCheck.instance.getToken(true);
-      //     expect(token, isA<String?>());
-      //   },
-      //   // TODO why is this Android/Web only?
-      //   skip: defaultTargetPlatform == TargetPlatform.iOS ||
-      //       defaultTargetPlatform == TargetPlatform.macOS,
-      // );
-
-      // test(
-      //   'setTokenAutoRefreshEnabled',
-      //   () async {
-      //     await expectLater(
-      //       FirebaseAppCheck.instance.setTokenAutoRefreshEnabled(true),
-      //       completes,
-      //     );
-      //   },
-      // );
-
-      // test('tokenChanges', () async {
-      //   final stream = FirebaseAppCheck.instance.onTokenChange;
-      //   expect(stream, isA<Stream>());
+      test('activate', () async {
+        await expectLater(
+          FirebaseAppCheck.instance.activate(
+            webRecaptchaSiteKey: '6Lemcn0dAAAAABLkf6aiiHvpGD6x-zF3nOSDU2M8',
+          ),
+          completes,
+        );
+      });
 
-      //   // TODO how to trigger event listener in e2e tests?
-      //   // await FirebaseAppCheck.instance.getToken(true);
-      //   //
-      //   // final result = await stream.first;
-      //   //
-      //   // expect(result, isA<AppCheckTokenResult>());
-      //   // expect(result.token, isA<String>());
-      // });
+      test(
+        'getToken',
+        () async {
+          final token = await FirebaseAppCheck.instance.getToken(true);
+          expect(token, isA<String>());
```

Comment:
```diff
+ expect(token, isA<String>());
- expect(token, isA<String?>());
```

In the commented out test was a check for `String?`. But since we expect a token that should not be null.